e2e/utils/TestUtils.ts

```diff
@@ -257,6 +317,7 @@ export async function startRancherDesktop(testPath: string, options?: { tracing?
     ],
     env: {
       ...process.env,
+      ...options?.env ?? {},
       RD_LOGS_DIR: reportAsset(testPath, 'log'),
       ...options?.mock ?? true ? { RD_MOCK_BACKEND: '1' } : {},
```

Comment:
I know it's existing code, but now that more tests are calling this function, and the majority are not mocked, maybe we should make the default `false`, and then you can just do `options?.mock ? { RD_MOCK_BACKEND: 1 }: {}`

Reply:
The default is mocked to encourage use of the mock backend (which is much faster); the extra effort required to use the non-mocked backend is intentional.